site/_docs/reference.md

```diff
@@ -1992,6 +1993,24 @@ Note:
 | value IS JSON ARRAY                               | Whether *value* is a json array, *value* is in character string type
 | value IS NOT JSON ARRAY                           | Whether *value* is not a json array, *value* is in character string type
 
+#### Other Operators
```

Comment:
Could you please make it clear that the operator `JSON_TYPE` is from MySQL?
My suggestion is to change "Other Operators" to "MySQL Specific Operators"